x-pack/plugins/ml/public/application/model_management/test_models/utils.ts

```diff
@@ -15,6 +15,14 @@ import type { ModelItem } from '../models_list';
 
 const PYTORCH_TYPES = Object.values(SUPPORTED_PYTORCH_TASKS);
 
+export function isDfaTrainedModel(modelItem: ModelItem) {
```

Comment:
This could be more like a type guard, for example:

```ts
export function isDfaTrainedModel(arg: unknown): arg is ModelItem) {
  ...
}
```

Reply:
🤔 I don't think it needs to be a type guard  since all the items are ModelItem types - it's more checking that certain optional properties exist.